cin_validator/rules/cin2022_23/rule_8620.py

```diff
@@ -0,0 +1,111 @@
+from typing import Mapping
+
+import pandas as pd
+
+from cin_validator.rule_engine import (
+    CINTable,
+    IssueLocator,
+    RuleContext,
+    rule_definition,
+)
+from cin_validator.test_engine import run_rule
+from cin_validator.utils import make_census_period
+
+# Get tables and columns of interest from the CINTable object defined in rule_engine/__api.py
+# Replace ChildIdentifiers with the table name, and LAChildID with the column name you want.
+
+CINdetails = CINTable.CINdetails
+CINclosureDate = CINdetails.CINclosureDate
+Header = CINTable.Header
+ReferenceDate = Header.ReferenceDate
+
+# define characteristics of rule
+@rule_definition(
+    code=8620,
+    module=CINTable.CINdetails,
+    message="CIN Closure Date present and does not fall within the Census year",
+    affected_fields=[CINclosureDate,ReferenceDate],
```

Comment:
So, reference date is relevant to the rule, but we don't count it as an affected field because this rule isn't really checking if it's wrong.